### PR TITLE
Implement Package section

### PR DESCRIPTION
**Pull request by JennyDoJD**
**Design**
Mobile
image
Desktop
image
**Result**
Mobile
image
image
Tablet
image
Desktop
image

### CODE REVIEW

File: practice/src/css/sections/package.css [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  
+  &:hover {
+    border: 4px solid var(--primary-border-radius-color);
+    transition: transform 0.3s ease;
+    transform: scale(0.98);
+    padding: 5px;
+  }
+}
+  
+.package-infor {
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/package.css]: Should be `info` or `information`. Not `infor`

File: practice/src/css/sections/package.css [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  line-height: var(--primary-line-height);
+}
+
+.btn-package:hover {
+  color: var(--tertiary-text-color);
+  background-color: var(--primary-background-color);
+}
+
+.package-secondary-title {
+  width: 223px;
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/package.css]: No need to set width here

File: practice/src/css/sections/package.css [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  }
+  
+  .package-card {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(200px, auto));
+    gap: 20px;
+  }
+
+  .package-secondary-title {
+    width: 100%;
```
Comment (huyencao-agilityio) [inline practice/src/css/sections/package.css]: No need to set width here

File: practice/src/index.html [hunk 4]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
       </div>
     </section>
+
+    <section class="package-section">
+      <!-- Package section -->
+      <div class="container wrapper-package">
+        <div class="package-card">
+          <div class="package-card-item">
+            <div class="package-card-image">
+              <img src="../src/assets/images/package/semeru.png" alt="Semeru Mountain" class="package-card-item-image">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should use jpeg or wepb instead of png

File: practice/src/index.html [hunk 5]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+              <a href="javascrip:void(0)" class="text package-title">Bromo</a>
+              <div class="package-address">
+                <img src="../src/assets/images/icon-subtract.svg" class="icon-subtract">
+                <p class="text package-card-desc">Malang, Indonesia</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="package-introduction">
+          <hgroup class="text package-heading">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Remove class `package-heading` if you don't use it to style

File: practice/src/index.html [hunk 6]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+            <h3 class="main-title">Our Package</h3>
+            <h4 class="secondary-title package-secondary-title">Popular Trip Packages</h4>
+          </hgroup>
+          <p class="text description introduction-desc">The packages available include accommodation for travel needs and insurance.</p>
+          <div class="text package-evaluation">
+            <div class="evaluation-list">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: remove class `evaluation-list` if you don't use it to style

File: practice/src/index.html [hunk 7]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+                <img src="../src/assets/images/icon-subtract.svg" class="icon-subtract">
+                <p class="text package-card-desc">Malang, Indonesia</p>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="text package-introduction">
+          <hgroup>
+            <h3 class="main-title">Our Package</h3>
+            <h4 class="secondary-title package-secondary-title">Popular Trip Packages</h4>
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: remove class `package-secondary-title` if you don't use it

File: practice/src/css/sections/package.css [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+.package-desc {
+  color: var(--gray-text-color);
+}
+
+.package-title {
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-medium);
+  line-height: var(--secondary-line-height);
+  color: var(--primary-text-color);
+  display: flex;
```
Comment (ghost) [inline practice/src/css/sections/package.css]: @JennyDoJD I am not sure why package-title should be `display: flex` ? could you help explain?

File: practice/src/css/sections/package.css [hunk 9]
```diff
@@ -0,0 +1,111 @@
...
+  line-height: var(--primary-line-height);
+}
+
+.btn-package:hover {
+  color: var(--tertiary-text-color);
+  background-color: var(--primary-background-color);
+}
+
+.package-card {
+  display: grid;
```
Comment (ghost) [inline practice/src/css/sections/package.css:86]: do you use both `flex` and `gird`? could you help detail the reasons for this?

File: practice/src/css/sections/package.css [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+/* Tablet */
+@media (min-width: 768px) {
+  .package-card {
+    grid-template-columns: repeat(auto-fit, minmax(200px, auto));
+  }
+}
+  
+/* Desktop */
+@media (min-width: 1440px) {
+  .package-introduction {
```
Comment (ghost) [inline practice/src/css/sections/package.css:99]: @JennyDoJD I am not sure why we need `width: 410px`

File: practice/src/css/sections/package.css (hunk 8 above)
Comment (JennyDoJD) [reply practice/src/css/sections/package.css]: I added it in the wrong place

File: practice/src/css/sections/package.css (hunk 10 above)
Comment (JennyDoJD) [reply practice/src/css/sections/package.css:99]: I want introduction-desc break down the line like design

File: practice/src/css/sections/package.css (hunk 9 above)
Comment (JennyDoJD) [reply practice/src/css/sections/package.css:86]: I only use grid for package-card

File: practice/src/index.html [hunk 11]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+              </div>
+              <a href="javascrip:void(0)" class="text package-title">Semeru Mountain</a>
+              <div class="package-address">
+                <img src="../src/assets/images/icon-subtract.svg" class="icon-subtract">
+                <p class="text package-card-desc">Malang, Indonesia</p>
+              </div>
+            </div>
+          </div>
+          <div class="package-card-item">
+            <div class="package-card-image">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should use figure instead div

File: practice/src/index.html [hunk 12]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
         </div>
       </div>
     </section>
+
+    <section class="package-section">
+      <!-- Package section -->
+      <div class="container wrapper-package">
+        <div class="package-card">
+          <div class="package-card-item">
+            <div class="package-card-image">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should use figure instead div

File: practice/src/index.html [hunk 13]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+              <div class="package-icon">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">                
+              </div>
+              <a href="javascrip:void(0)" class="text package-title">Bromo</a>
+              <div class="package-address">
+                <img src="../src/assets/images/icon-subtract.svg" class="icon-subtract">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should add alt for image

File: practice/src/index.html [hunk 14]
```diff
@@ -70,6 +70,77 @@ <h5 class="service-topic-title">Easy trip booking</h5>
...
+              <div class="package-icon">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+                <img src="../src/assets/images/icon-star.svg" alt="Icon star" class="icon-star">
+              </div>
+              <a href="javascrip:void(0)" class="text package-title">Semeru Mountain</a>
+              <div class="package-address">
+                <img src="../src/assets/images/icon-subtract.svg" class="icon-subtract">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Should add alt for image

**Pull request merged**